tuplan_garage/planning/training/modeling/models/pdm_offset_model.py

```diff
@@ -144,12 +157,18 @@ def forward(self, features: FeaturesType) -> TargetsType:
         #     [state_encodings, centerline_encodings, trajectory_encodings], dim=-1
         # )
 
-        input_embeds, info_dict = self._model.encoder(is_training=False, **input)
+        input_embeds, info_dict = self._model.encoder(is_training=self.training, **kwargs)
```

Comment:
use **input, incase input has more keys in the future

Reply:
I copy all key-value pairs from input to kwargs because encoder can't handle a PDMFeature as inputs